user/services/UserManager.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import User #this will be replaced by our class user
+from django.contrib.auth import authenticate, login as log, logout
+
+def register(username,firstname,lastname, email, passwd):
+    user = User.objects.create_user(username, email= email, password=passwd)
+    user.first_name = firstname
```

Comment:
Ok, but you can try it, I think it might work: in the documentation, they state that all keyword arguments (**extra_fields) are passed to the User __init__ function and User has first_name and last_name fields. But it's not important, we were just curious what is the reason :-)